Data Structures/Linked Lists/C++/Doubly_Linked_List.cpp

```diff
@@ -0,0 +1,91 @@
+#include <iostream>
+
+using namespace std;
```

Comment:
Please remove this line. It is not good practice to pollute the global namespace.